pages/1.9/release-notes/1.9.8/index.md

```diff
@@ -0,0 +1,235 @@
+
+For more information, see the debugging [documentation](/1.9/monitoring/debugging/).
+
+[preview]
+### Logging
+[/preview]
+
+Stream task and system logs to journald by setting the `mesos_container_log_sink` install-time parameter to `journald` or `journald+logrotate`. This allows you to:
+
+- Include task metadata like container ID in your queries to more easily locate the logs that you want.
+- Use the new DC/OS CLI commands `dcos node log` and `dcos task log` to query logs. You can also make HTTP requests directly against the new Logging API.
+- Set up log aggregation solutions such as Logstash to get logs into their aggregated storage solutions.
+
+For more information, see the [documentation](/1.9/monitoring/logging/).
+
+[preview]
+### Metrics
+[/preview]
+
+- Node-level HTTP API that returns metrics from tasks, cgroup allocations per container, and host level metrics such as load and memory allocation.
+- StatsD endpoint in every container for forwarding metrics to the DC/OS metrics service. This service is what exposes the HTTP API.
+- Any metric sent to STATSD_UDP_HOST/PORT is available in the HTTP API’s `/container/<container_id>/app` endpoint.
+
+For more information, see the [documentation](/1.9/metrics/).
+
+### Tool for Troubleshooting Service Deployment Failures
+
+- The new service deployment troubleshooting tool allows you to find out why your applications aren’t starting from the GUI and CLI.
+
+  ![Service deploy GUI](/1.9/img/dcos-offers.png)
+
+### Improved GUI
+
+- New look and feel and improved navigation.
+
+  ![New GUI](/1.9/img/dcos-dash.png)
+
+- Usability improvements to the service create workflow.
+
+  ![Improved GUI](/1.9/img/dcos-create.png)
+
+## Networking Services
+
+- CNI support for 3rd party CNI plugins.
+- Performance improvements across all networking features.
+
+## Security and Governance
+
+- DC/OS Identity and Access Management (IAM) highlights:
+
+  - LDAP group import: support importing `posixGroup` objects according to [RFC2307](https://www.ietf.org/rfc/rfc2307) and [RFC2307bis](https://tools.ietf.org/html/draft-howard-rfc2307bis-02), and ensure compatibility with [FreeIPA](https://www.freeipa.org) and [OpenLDAP](https://www.openldap.org/).
+  - SAML 2.0: ensure that the authentication flow works against [Shibboleth](https://www.shibboleth.net/products/identity-provider/) and improve compatibility with a wide range of identity provider configurations.
+  - OpenID Connect: ensure that the authentication flow works against [dex](https://github.com/coreos/dex) as well as against [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/), and allow for customizing the identity provider certificate verification in back-channel communication. Enhance configuration validation for a better user experience.
+
+- DC/OS CLI highlights:
+
+  - Support single sign-on authentication via OpenID Connect and SAML 2.0 against the DC/OS IAM.
```

Comment:
Maybe Enterprise: If so, add [enterprise type="inline" size="small" /]

Reply:
@masonse, @jieyu, @somikbehera, @jgehrcke, @keithchambers, @ToddGreenstein, could you confirm if this section is for Enterprise?

Reply:
@somikbehera could you please review the section "Security and Governance" and confirm if I need to add ENTERPRISE tags or not?

Reply:
All of the things mentioned in the section `Security and Governance` are enterprisy. I double-checked this for every single line in this section.

Reply:
Added Enterprise tags for each feature in "Security and Governance" section.